src/testRunner/unittests/reuseProgramStructure.ts

```diff
@@ -305,10 +305,10 @@ namespace ts {
             assert.equal(program1.structureIsReused, StructureIsReused.Not);
         });
 
-        it("fails if rootdir changes", () => {
+        it("succeeds if rootdir changes", () => {

```

Comment:
I think this is correct? `rootDir` doesn't affect module resolution, it affects emit.